app/views/onboarding/success.html.erb

```diff
@@ -1,4 +1,5 @@
-<%= c 'onboarding_success',
-  title: Setting.onboarding_success_title,
-  text: Setting.onboarding_success_text
+<%= c 'onboarding_response',
+  style: :success,
+  heading: I18n.t('components.onboarding_success.heading', project_name: Setting.project_name),
+  text: I18n.t('components.onboarding_success.text').html_safe
 %>
```

Comment:
I think this is a regression, as editors won’t be able to update the UI texts for the onboarding page using the new settings UI. 

```suggestion
<%= c 'onboarding_response',
  style: :success,
  heading: Setting.onboarding_success_title,
  text: ISetting.onboarding_success_text
%>
```

Reply:
good catch